jobOptions.mac

```diff
@@ -15,4 +15,17 @@
 /WCSim/physics/list WCSim # included for backward-compatibility.  Requires jobOptions2.mac
 
 
-
+#choose the Trigger & Digitizer type (and options)
+# set at least one (both to be sure) to SKI_SKDETSIM if you want to use the old combined Digitization & Trigger classes
```

Comment:
Does this mean set either /DAQ/Digitizer or /DAQ/Trigger to SKI_SKDETSIM? This comment could maybe be clearer.